apps/_pomodoro_app/realtime/pomodoroSocketIOServer.js

```diff
@@ -0,0 +1,94 @@
+const Timer = require('./application/Timer')
+const SimpleAuthMiddleware = require('./application/SimpleAuthMiddleware')
+const ConsoleLogger = require('./application/ConsoleLogger')
+const Repository = require('./application/Repository')
+
+
+/**
+ * Realtime server that handles the pomodoro timer application
+ * TODO When socket dies we fire out an logger request to create statistics
+ * @param io - Socket.io namespace
+ * @param logger - an AbstractLogger impl
+ * @param auth - an AbstractAuthMiddleware impl
+ * @param repo - an AbstractRepository impl
+ */
+function init (io, logger=new ConsoleLogger(), auth=SimpleAuthMiddleware, repo=new Repository()) {
```

Comment:
it's better to set `args = null` and then create it if needed

Reply:
Fixed, now we don't need complex IoC